recipes/suitesparse/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "4.4.5" %}
+
+{% set variant = "openblas" %}
+
+
+package:
+  name: suitesparse
+  version: {{ version }}
+
+source:
+  fn: SuiteSparse-{{ version }}.tar.gz
+  url: http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-{{ version }}.tar.gz
+  sha256: 83f4b88657c7dc57681633e8ca6835ddb12c146bc51af77b6494972ed1ea8bc9
+
+build:
+  skip: true  # [win]
+  number: 200
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - tbb
+
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - tbb
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libamd.a"
+    - test -f "${PREFIX}/lib/libbtf.a"
+    - test -f "${PREFIX}/lib/libcamd.a"
+    - test -f "${PREFIX}/lib/libccolamd.a"
+    - test -f "${PREFIX}/lib/libcholmod.a"
+    - test -f "${PREFIX}/lib/libcolamd.a"
+    - test -f "${PREFIX}/lib/libcxsparse.a"
+    - test -f "${PREFIX}/lib/libklu.a"
+    - test -f "${PREFIX}/lib/libldl.a"
+    - test -f "${PREFIX}/lib/librbio.a"
+    - test -f "${PREFIX}/lib/libspqr.a"
+    - test -f "${PREFIX}/lib/libsuitesparseconfig.a"
+    - test -f "${PREFIX}/lib/libumfpack.a"
+
+about:
+  home: http://faculty.cse.tamu.edu/davis/suitesparse.html
+  license: BSD 3-Clause or LGPL v2.1, LGPL v2.1, GPL v2 (UMFPACK)
+  summary: A suite of sparse matrix algorithms
+
+extra:
+  recipe-maintainers:
+    - grlee77
```

Comment:
Added you as a maintainer, @grlee77, based on your interest in SuiteSparse. Hope that is ok.